### PR TITLE
[Vue] Basic stopwatch app with Vuejs

### LINKED ISSUE #370: [New]: Stopwatch with Vuejs
### Framework/code-stack

Vue

### Description

Stopwatch with Vuejs 

### Similar challenges

No

### Screenshots

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by Shardik04**
# Title and Issue number 

Title : Stopwatch with Vuejs

Issue No. : #370 

Code Stack : Vuejs

Close #370

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [] I have created a helpful and easy to understand `README.md`
- [] I have updated the Index.html file for my contribution

### CODE REVIEW

File: apps/vue/src/challenges/stopwatch/index.vue [hunk 1]
```diff
@@ -0,0 +1,121 @@
+<template>
+    <div class="stopwatch">
+        <h5 class="flex-center">Stopwatch</h5>
```
Comment (sadanandpai) [inline apps/vue/src/challenges/stopwatch/index.vue:3]: heading is not needed

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Comment by sadanandpai** [comment]
image

contributor icon is broken.

File: apps/vue/src/challenges/stopwatch/index.vue [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+
+    const { hour = 0, min = 0, sec = 0 } = time.value;
+    return !(hour > 0 || min > 0 || sec > 0);
+
+}
+
+</script>
+  
+<style scoped css="scss">
```
Comment (arpansaha13) [inline apps/vue/src/challenges/stopwatch/index.vue:94]: I think the `css="scss"` property is not required. See if it works without this property.

File: apps/vue/src/challenges/stopwatch/index.vue [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+const isResetDisabled = () => {
+    if (!intervalID) {
+        return true
+    }
+
+
+    const { hour = 0, min = 0, sec = 0 } = time.value;
+    return !(hour > 0 || min > 0 || sec > 0);
+
+}
```
Comment (arpansaha13) [inline apps/vue/src/challenges/stopwatch/index.vue:90]: You may try computed properties for this function

File: apps/vue/src/challenges/stopwatch/index.vue [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    min: 0,
+    sec: 0,
+});
+
+
+const intervalID = ref(0);
+
+// Local state value update
+const updateTimer = () => {
+    time.value = { ...time.value };
```
Comment (arpansaha13) [inline apps/vue/src/challenges/stopwatch/index.vue:31]: Please check if the functionality still works if this line is removed

**Review by arpansaha13 — CHANGES_REQUESTED** [review]

**Review by arpansaha13 — APPROVED** [review]

**Pull request merged**